cloud/hack/update-codegen.sh

```diff
@@ -22,5 +22,5 @@ SCRIPT_ROOT=$(unset CDPATH && cd $(dirname "${BASH_SOURCE[0]}")/../.. && pwd)
 
 ${SCRIPT_ROOT}/cloud/hack/generate-groups.sh "deepcopy,client,informer,lister" \
 github.com/kubeedge/kubeedge/cloud/pkg/client github.com/kubeedge/kubeedge/cloud/pkg/apis \
-"devices:v1alpha2 reliablesyncs:v1alpha1" \
+"devices:v1alpha2 reliablesyncs:v1alpha1 rules:v1" \
```

Comment:
How about using `v1alpha1`, I think we still need some work to achieve GA.

Reply:
I think the function of rules is stable, we can use v1.